nixos/doc/manual/configuration/linux-kernel.chapter.md

```diff
@@ -102,45 +102,15 @@ pkgs.linux_latest.override {
 ```
 
 See `pkgs/os-specific/linux/kernel/generic.nix` for details on how these arguments
-affect the generated configuration. You can also build a custom version of Linux by calling
-`pkgs.buildLinux` directly, which requires the `src` and `version` arguments to be specified.
+affect the generated configuration.
+You can also use any of the linux builders directly (see [this section](https://nixos.org/nixpkgs/manual#sec-linux-kernel) on the Nixpkgs manual), which will likely require the `src` and `version` arguments to be specified.
 
 To use your custom kernel package in your NixOS configuration, set
 
 ```nix
 boot.kernelPackages = pkgs.linuxPackagesFor yourCustomKernel;
 ```
 
-Note that this method will use the common configuration defined in `pkgs/os-specific/linux/kernel/common-config.nix`,
-which is suitable for a NixOS system.
-
-If you already have a generated configuration file, you can build a kernel that uses it with `pkgs.linuxManualConfig`:
-
-```nix
-let
-  baseKernel = pkgs.linux_latest;
-in pkgs.linuxManualConfig {
-  inherit (baseKernel) src modDirVersion;
-  version = "${baseKernel.version}-custom";
-  configfile = ./my_kernel_config;
-  allowImportFromDerivation = true;
-}
-```
-
-::: {.note}
-The build will fail if `modDirVersion` does not match the source's `kernel.release` file,
```

Comment:
This seems now gone. If it's still valid we should keep it in the appropriate context.

Reply:
It's a slightly lower-level version of the example I added in the Nixpkgs section. I'll port the details regarding `modDirVersion` to keep it "backwards compatible", since some folks might have copied this example already.